boards/arm/xmc4/xmc4700-relax/src/xmc4_bringup.c

```diff
@@ -26,9 +26,15 @@
 
 #include <debug.h>
 #include <errno.h>
-#include <nuttx/spi/spi_transfer.h>
 #include <sys/types.h>
 
+#ifdef CONFIG_XMC4_USCI_SPI
+  #include <nuttx/spi/spi_transfer.h>
+#endif
+#ifdef CONFIG_USERLED
+  #include <nuttx/leds/userled.h>
```

Comment:
```
#  include <nuttx/leds/userled.h>
```

Reply:
@trns1997 as @trns1997 the correct is "#  " instead "  #".
First time I saw it I was surprised because I didn't know that the compiler was able to recognize space after # as valid directive.

Reply:
Alright noted, sorry just my usual coding habit. It is more readable your way though I guess I might take it up :)

Reply:
Don't worry, @trns1997 but again, it's better to amend the style change to the first patch.

Reply:
Sorry, I missed this comment, in fact, he left some minor issues in the first commit "# define" instead of "#  define".
@trns1997 when you submit a new PR to XMC4700, please remember to fix those modificações you did in the source code